test/factories/admins.rb

```diff
@@ -1,8 +0,0 @@
-FactoryGirl.define do
-  factory :admin do
-    username "MyString"
-    password_digest "MyString"
-    email "MyString"
-  end
-
-end
```

Comment:
I am putting all of the factories inside of the factories.rb file in the factories folder. I do not want us to have separate factory files inside of the factory folder. I'd rather us have just one factories.rb file.